martin-mbtiles/src/tile_copier.rs

```diff
@@ -28,9 +28,12 @@ pub struct TileCopierOptions {
     /// Maximum zoom level to copy
     #[cfg_attr(feature = "cli", arg(long, conflicts_with("zoom_levels")))]
     max_zoom: Option<u8>,
-    /// List of zoom levels to copy; if provided, min-zoom and max-zoom will be ignored
+    /// List of zoom levels to copy
     #[cfg_attr(feature = "cli", arg(long, value_parser(ValueParser::new(HashSetValueParser{})), default_value=""))]
     zoom_levels: HashSet<u8>,
+    /// If provided this file will be used to generate a diff with the source file
```

Comment:
How about this text?  I don't think we need "if provided" because these are options already?

```suggestion
    /// Compare source file with this file, and only copy non-identical tiles to destination
```